docker-integrations/tendermint-python/.gitignore

```diff
@@ -0,0 +1,192 @@
+# Created by https://www.toptal.com/developers/gitignore/api/node
+
+# Edit at https://www.toptal.com/developers/gitignore?templates=node
+
+### Node
+
+# Logs
+
+logs
+_.log
+npm-debug.log_
```

Comment:
Same here, this `.gitignore` specifically target Node and Javascript applications. This might be more relevant https://www.toptal.com/developers/gitignore/api/python.

Reply:
Updated `.gitignore` with the one proposed